builder/builder.go

```diff
@@ -1362,6 +1362,14 @@ func (b *Builder) BuildChroots(template *x509.Certificate, privkey *rsa.PrivateK
 		}
 	}
 
+	// Generate the certificate needed for signing verification if it does not exist
```

Comment:
Optional Comment: This  move seems completely random, looks better next to the block that installs it.

Reply:
Actually the move is not completely random. The cert needs to be generated before the bundles are built so it can be optionally put into the os-core-update bundle and the full chroot. I did modify the comment because I'm not doing the insertion at this point.

Reply:
:+1: